features/freaks/src/main/kotlin/com/agilefreaks/freaks_catalog/features/freaks/FreaksFragment.kt

```diff
@@ -111,9 +111,8 @@ class FreaksFragment : Fragment() {
         mBottomSheetBinding.viewModel = filterViewModel
         mBottomSheetBinding.filterTitle.text = name
 
-        val recyclerFiltersView =
-            mBottomSheetBinding.recyclerFiltersView
-        recyclerFiltersView.layoutManager = LinearLayoutManager(requireContext())
+        mBottomSheetBinding.recyclerFiltersView.layoutManager =
```

Comment:
Why are we naming variables with m_____? 
I like a lot this comment about this subject: https://stackoverflow.com/a/26563953/9648176
What are your thoughts about this?